data/ideas.json

```
@@ -185,4 +185,10 @@
         "level": "advanced",
         "tags": "roadmap, success, process"
     }
+    {
+        "title": "Clash of Space",
+        "description": "This is space-craft and alien shooting game where a player can earn points based on how many alien spacecrafts the player can destroy and the player has to protect his/her spaceship from the aliens to save the health level.",
+        "level": "basic",
+        "tags": "clas-of-space, game, spacewar, alien-shooting, action"
```

Please separate the word using the dash(`-`)
```suggestion
        "tags": "clash-of-space, game, space-war, alien-shooting, action"
```